Core/Core/AppEnvironment/K5State.swift

```diff
@@ -47,5 +47,6 @@ public class K5State {
 
     public func userDidLogout() {
         isK5Account = false
+        AppEnvironment.shared.userDefaults?.isK5StudentView = false
```

Comment:
It would be better to write the local `sessionDefaults` variable here, they share the same instance but if we already have DI introduced let's use it.